### ISSUE TITLE
npx typewriter init generates yml file, but not client (Windows)

### ISSUE THREAD

**Issue by RapidOwl** [segmentio/typewriter#318]
Hi folks! We've been using for Segment for several years, but never used Typewriter until now. I'm having a bit of trouble getting it set up.

TL;DR `npx typewriter init` generates the `typewriter.yml`, but no client.

I've tried this in Git Bash, Powershell and Command Prompt, but the only one that let you use arrow keys to select the options was the Command Prompt.

Dependency versions:
```
"@segment/analytics-next": "^1.66.0",
"typewriter": "^9.2.0"
```

Steps to reproduce:
1. Install dependencies as per [docs](https://segment.com/docs/protocols/apis-and-extensions/typewriter/#browser-quickstart).
2. Run `npx typewriter init`.
3. Following it through, setting `typewriter` as the folder name, JavaScript as the language and the web SDK.
4. Look in the project folder for the client.

Here's a screenshot of my terminal:
![image](https://github.com/segmentio/typewriter/assets/1280948/3360b591-7b04-42bb-8f9f-8d0f065cc654)

Here's a screenshot of my tracking plan (I'm able to receive events in the Segment debugger and generate violations with incorrect values):
![image](https://github.com/segmentio/typewriter/assets/1280948/0cf9cdb6-a5c6-4f80-b6a2-43c9bfa9a744)

Thanks for any help anyone can offer!

**Comment by JAjedvalmar** [comment]
hey @RapidOwl ! have you tried going through the advance settings from the `npx typewriter init` command? I remember we faced a similar issue and just defining those extra options in the yml file did the trick for the client to be generated, you can leave a bunch of those options like empty strings but i think it might help your case. Just to clarify, the advanced options appear after you've chosen the SDK (the [quicktype.io](https://app.quicktype.io/) options).

**Comment by cideM** [comment]
I am seeing the same on Darwin using Typewriter 9.2.0.

Going through the [code of the `initCommand`](https://github.com/segmentio/typewriter/blob/master/src/commands/init.ts) it doesn't look like anything actually creates any clients.

`update` also doesn't, but running just `typewriter` later on does generate some files.
